lib/manager/components/deployment/PeliasPanel.js

```diff
@@ -175,6 +222,7 @@ class PeliasPanel extends Component<Props, State> {
         >
           Download
         </Button>
+        {label && <em style={{marginLeft: '1ch'}}>{label}</em>}
```

Comment:
The text seems to need a `vertical-align: middle` in Firefox, but that's nothing serious

Reply:
Good catch thanks!